_drafts/yyyy-mm-dd-game-template.html

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Did you mean to commit this file? What it fo'?

Reply:
I did, but now I'm like…is it useful? probably not, but I don't feel strongly 1 way or the other